src/main/resources/assets/splatcraft/lang/it_it.json

```diff
@@ -495,26 +539,34 @@
+  "gamerule.splatcraft.inkHealingConsumesHunger": "La rigenerazione nell'inchiostro consuma fame",
+  "gamerule.splatcraft.inkHealingConsumesHunger.description": "Fa in modo che il recupero vita nell'inchiostro richieda e consumi la barra della fame",
   "gamerule.splatcraft.dropCrateLoot": "Rilascia il contenuto delle scatole quando distrutte",
   "gamerule.splatcraft.requireInkTank": "Richiedi il serbatoio per le armi",
+  "gamerule.splatcraft.requireInkTank.description": "Determina se le armi della Splatcraft necessitino l'inchiostro di un serbatoio oppure no",
   "gamerule.splatcraft.universalInk": "Inchiostro universale",
-  "gamerule.splatcraft.universalInk.description": "Permette ai giocatori di nuotare nell'inchiostro nemico.",
+  "gamerule.splatcraft.universalInk.description": "Permette ai giocatori di nuotare nell'inchiostro nemico",
   "gamerule.splatcraft.inkFriendlyFire": "Fuoco amico colore",
   "gamerule.splatcraft.inkFriendlyFire.description": "Le armi a inchiostro possono danneggiare giocatori con lo stesso colore.",
   "gamerule.splatcraft.inkDamageCooldown": "Annulla i danni da inchiostro alla rigenerazione",
-  "gamerule.splatcraft.inkDamageCooldown.description": "Impedisce ai danni da inchiostro di ignorare il tempo di invincibilità.",
+  "gamerule.splatcraft.inkDamageCooldown.description": "Impedisce ai danni da inchiostro di ignorare il tempo di invincibilità",
   "gamerule.splatcraft.inkMobDamagePercentage": "Percentuale danni contro i mob",
   "gamerule.splatcraft.inkMobDamagePercentage.description": "Determina quanto moltiplicare il danno da inchiostro contro i mob",
   "gamerule.splatcraft.infiniteInkInCreative": "Inchiostro infinito in creativa",
   "gamerule.splatcraft.infiniteInkInCreative.description": "I giocatori in creativa hanno inchiostro infinito",
   "gamerule.splatcraft.inkableGround": "Le armi Splatcraft possono colorare terreno",
-  "gamerule.splatcraft.inkableGround.description": "Permette alle armi Splatcraft di colorare i blocchi ovunque",
+  "gamerule.splatcraft.inkableGround.description": "Permette alle armi Splatcraft di colorare i blocchi ovunque nel mondo",
   "gamerule.splatcraft.rechargeableInkTank": "Serbatoio ricaricabile",
-  "gamerule.splatcraft.rechargeableInkTank.description": "Permette ai serbatoi di ricarsi passivamente, più velocemente mentre si è immersi.",
+  "gamerule.splatcraft.rechargeableInkTank.description": "Permette ai serbatoi di ricarsi passivamente, più velocemente mentre si è immersi",
+  "gamerule.splatcraft.globalSuperJumping": "Super salto globale",
+  "gamerule.splatcraft.globalSuperJumping.description": "Permette ai giocatori di eseguire super salti fuori dagli scenari",
+  "gamerule.splatcraft.superJumpDistanceLimit": "Limite di distanza super salto",
+  "gamerule.splatcraft.superJumpDistanceLimit.description": "Determina la massima distanza di super salto al di fuori degli scenari. §oI valori negativi disabilitano interamente il limite",
```

Comment:
`§o` isn't present in English, is there any reason it's here?

Reply:
`§o` just makes the text in italic; I'll remove it if you think it's unnecessary